libdokan/createfile_test.go

```diff
@@ -0,0 +1,46 @@
+package libdokan
+
+import (
+	"os"
+	"syscall"
+)
+
+func Open(filename string) (*os.File, error) {
```

Comment:
They are 1:1 to os package functions, thus want to keep the names the same. As this is a _test file they won't be exported.